Add key definitions for seats and collections (#800)
directus/docs: content/guides/03.auth/2.access-control.md

## Studio Users

Studio Users are the users counted against the seat limit on your plan, whether you are self-hosted or on Directus Cloud. A Studio user meets at least one of the following criteria:

1. At least one policy with 'Admin Access' enabled.
2. At least one policy with 'App Access' enabled, allowing access to the Data Studio.

> Users who have neither are API-only users and do not consume a seat, however many of them you have. See [Key Definitions](/licensing/overview#key-definitions) in the licensing overview for the full seat and collection counting rules.

Reviewer: This first sentence is hard to read.

```suggestion
Users with neither are considered API-only users and do not consume a seat, regardless of how many of them may exist. See [Key Definitions](/licensing/overview#key-definitions) in the licensing overview for the full seat and collection counting rules.
```
